tests/test_odd_even.py

```diff
@@ -17,7 +19,7 @@ def test_odd_even():
 
     assert_allclose(odd[0], 0.5)
     assert_allclose(even[0], 0.5)
-    assert np.isnan(sigma)
+    assert sigma == 0.0
```

Comment:
When it comes to float comparison, `assert_allclose` is more robust, but I'll leave it to your good judgement.
```suggestion
    assert_allclose(sigma, 0.0)
```

Reply:
Good call.